google/cloud/sql/connector/instance.py

```diff
@@ -206,8 +220,10 @@ async def _refresh_task(self: RefreshAheadCache, delay: int) -> ConnectionInfo:
             """
             refresh_task: asyncio.Task
             try:
-                logger.debug(f"['{self._instance_connection_string}']: Entering sleep")
                 if delay > 0:
+                    logger.debug(
+                        f"['{self._instance_connection_string}']: Entering sleep"
```

Comment:
```suggestion
                        f"['{self._instance_connection_string}']: Background refresh will sleep for {delay / 60} minutes."
```

Reply:
I am going to just remove this log. It is essentially the same as the "Refresh operation scheduled in XX minutes" log message